infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesNamespaceFactory.java

```diff
@@ -179,6 +187,36 @@ KubernetesNamespace doCreateNamespaceAccess(String workspaceId, String name) {
         clientFactory, cheClientFactory, sharedPool.getExecutor(), name, workspaceId);
   }
 
+  @ScheduleRate(period = 1, initialDelay = 1, unit = TimeUnit.HOURS)
```

Comment:
imo printing warn on server startup is more efficient. Make sense consider changing delay to 0 ?

Reply:
this method is called in this class constructor. As a result of setting the delay to 0, we will see two identical messages almost at the same time.